src/App.tsx

```diff
@@ -1,29 +1,70 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+
+type Intial<T extends number[]> = {
```

Comment:
Typo in the type name 'Intial'. It should be 'Initial' to correctly represent the intention.